.cirrus.yml

```diff
@@ -87,6 +87,34 @@ qa_task:
     - jfrog rt npm-config --repo-resolve npm --repo-deploy sonarsource-npm-public-qa
     - git submodule update --init --depth 1 -r
     - jfrog rt npm-ci
+    - npm install eslint-ruling@npm:eslint@8.57.0
+    - npm run build
+    - npm run ruling
+
+qa_eslint9_task:
+  depends_on:
+    - build
+  eks_container:
+    matrix:
+      - docker_arguments:
```

Comment:
here we remove the test on Nodev16 (that we perform with eslint8), as eslint9 requires Nodev18.18